components/proxy/src/test/kotlin/com/hotels/styx/routing/handlers/BackendServiceProxyTest.kt

```diff
@@ -42,6 +43,8 @@ class BackendServiceProxyTest : StringSpec({
     val baRequest = LiveHttpRequest.get("/ba/x").build()
 
     val environment = Environment.Builder().build()
+    val context = RoutingContext(environment = environment).get()
```

Comment:
Maybe we should specify when we are going to use named parameters (see other methods below that would potentially benefit from them).

Reply:
Do you mean "specify when we are going to use ..." as in coding guidelines?